acceptance/acceptance_test.go

```diff
@@ -1756,6 +1756,46 @@ func testAcceptance(
 						})
 					})
 
+					when("--cache with options for build cache as image", func() {
```

Comment:
You'll want to _gate_ these tests to only execute when pack supports this new `--cache` flag. This is due to the fact that we use the same acceptance tests to run compatibility tests. IOW, these tests may be executed on older versions of pack in which case they will fail.

Here's an example of how the gating is typically done for another similar newish feature called `--creation-time`:

1. [Add a feature definition](https://github.com/buildpacks/pack/blob/c1e75c213f746592eb6963fdd9a69af463f6922c/acceptance/invoke/pack.go#L221-L227).
2. [Gate using `it.Before`](https://github.com/buildpacks/pack/blob/c1e75c213f746592eb6963fdd9a69af463f6922c/acceptance/acceptance_test.go#L1976-L1977).